app/uk/gov/hmrc/eusubsidycompliancefrontend/job/RemoveExchangeRateJob.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliancefrontend.job
+import play.api.Logging
+import play.api.inject.ApplicationLifecycle
+import uk.gov.hmrc.eusubsidycompliancefrontend.config.AppConfig
+import uk.gov.hmrc.eusubsidycompliancefrontend.persistence.MonthlyExchangeRateCache
+
+import javax.inject._
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class RemoveEmailJob @Inject() (
+  lifecycle: ApplicationLifecycle,
+  monthlyExchangeRateCache: MonthlyExchangeRateCache
+)(implicit
+  val appConfig: AppConfig,
+  executionContext: ExecutionContext
+) extends Logging {
+
+  def runExchangeRateRepoJob(): Unit = {
+    logger.warn("=== clearing ExchangeRate Repository job starting ===")
+    monthlyExchangeRateCache.deleteAll().map(_ => logger.warn(s"=== exchangeRate cache cleared ==="))
```

Comment:
I believe deleteAll does indeed just drop the collection already, but the naming is a tad confusing 😄

Reply:
Just for clarification, `deleteAll` drops and recreates it (which is the most efficient way of deleting all the records). If you want only to drop the collection, a better way to drop it would be by injecting `MongoComponent` instead of `MonthlyExchangeRateCache` and doing this:

```scala
mongoComponent.database.getCollection("exchangeRateMonthlyCache").drop().toFuture().void
```